avr-hal-generic/src/i2c.rs

```diff
@@ -609,6 +631,13 @@ macro_rules! impl_i2c_twi {
                     .write(|w| w.twen().set_bit().twint().set_bit().twsto().set_bit());
                 Ok(())
             }
+
+            #[inline]
+            fn raw_force_stop(&mut self) -> Result<(), Error> {
+                self.twcr
+                    .write(|w| w.twint().set_bit());
```

Comment:
I know it will be a no-op, but I think there is value in explicitly adding a 

```rust
.twen().clear_bit()
```

here to make it abundantly clear that `raw_force_stop()` works by completely disabling the peripheral.